package/com.unity.formats.usd/Dependencies/USD.NET/collections/SampleCollection.cs

```diff
@@ -29,6 +29,8 @@ public class SampleCollection<T> : IEnumerable<SampleEnumerator<T>.SampleHolder>
         private SdfPath[] m_paths;
         private Scene m_scene;
 
+        public int GetLength() => m_paths.Length;
```

Comment:
My reasoning for calling it 'GetLength' was so that it was explicit that it was retrieving something from a member, and not to be confused with an Array.Length. I might be overthinking this though. Do you think making it a property would be nicer?